.github/CONTRIBUTING.md

```diff
@@ -49,18 +49,30 @@ Please see the [Contributor Guide in `PowerShell/PowerShell-Docs`](https://githu
 
 #### Spellchecking documentation
 
-Documentation are spellchecked. We make use of the
+Documentation is spellchecked. We use the
 [markdown-spellcheck](https://github.com/lukeapage/node-markdown-spellcheck) command line tool,
 which can be run in interactive mode to correct typos or add words to the ignore list
 (`.spelling` at the repository root).
 
-To run the spellchecker, follow the steps as follows:
+To run the spellchecker, follow these steps:
 
 * install [Node.js](https://nodejs.org/en/) (v6.4.0 or up)
 * install [markdown-spellcheck](https://github.com/lukeapage/node-markdown-spellcheck) by
   `npm install -g markdown-spellcheck` (v0.11.0 or up)
 * run `mdspell "**/*.md" --ignore-numbers --ignore-acronyms --en-us`
-* if the `.spelling` file is updated, commit and push it
+* if the `.spelling` file is updated, commit and push it  
+
+#### Checking links in documentation  
+Documentation is link-checked. We make use of the
```

Comment:
I would suggest saying that this and markdown syntax checking (like the newline before and after headers.) can be checked by running `start-pspester -includecommontests`

Reply:
👍